src/languageservice/services/yamlCompletion.ts

```diff
@@ -797,12 +792,7 @@ export class YAMLCompletion extends JSONCompletion {
         break;
       case 'object':
         {
-          const objectInsertResult = this.getInsertTextForObject(
-            schema,
-            separatorAfter,
-            `${indent}${this.indentation}`,
-            insertIndex++
-          );
+          const objectInsertResult = this.getInsertTextForObject(schema, separatorAfter, `${this.indentation}  `, insertIndex++);
```

Comment:
Do you need to also make this change for line 736 above?

Reply:
No